README.md

```diff
@@ -36,7 +36,8 @@ Generates the following Ethereum addresses:
 * OTHERcoin address: 0x642D2B84A32A9A92FEc78CeAA9488388b3704898
 * UniswapAdaptor address: 0xE4eA76e830a659282368cA2e7E4d18C4AE52D8B3
 * UniswapFactory address: 0xd2D23b73A67208a90CBfEE1381415329954f54E2
-* NodeRegistry: 0xBFCF120a8fD17670536f1B27D9737B775b2FD4CF
+* Tracker NodeRegistry: 0xBFCF120a8fD17670536f1B27D9737B775b2FD4CF
```

Comment:
Question: Will the address of the tracker registry stay the same when [this change](https://github.com/streamr-dev/network-contracts/pull/5) is deployed? (ie. will it change "in place"?)

Reply:
yes, why wouldnt it? Address is hash(key + nonce).

Reply:
Ok, just confirming the breaking change, as it will probably cause some tests in `network` or the protocol repo to fail after this is merged.